### ISSUE TITLE
Debugger only resumes one nodejs project when multiple are running

### ISSUE THREAD

**Issue by macromaniac** [microsoft/nodejstools#1825]
##### Expected Behavior 

When I exit a breakpoint with multiple projects I expect all the projects to resume operation when I hit Continue (F5)

##### Actual Behavior 

only 1 project (the one in the top left of the bar) resumes.

##### Steps to Reproduce

Tested on Nodejs version 6.12.2 and 8.9.3

1) create two startup projects (I use typescript nodejs console projects)
2) have them print something out every 100 milliseconds
3) put a breakpoint on one of the print event loops
4) wait like 3-4 seconds (doesn't work if you do it immediately)
5) when you hit F5 again it will appear as if all projects are running, but only the active debugging project will have resumed (this is the one in the top left bar)

here is a test project

[testcounter.zip](https://github.com/Microsoft/nodejstools/files/1593106/testcounter.zip)

Joey

**Comment by paulvanbrenk** [comment]
This is by design, and works the same way as other project types in VS.

**Comment by macromaniac** [comment]
It doesn't work this way for C#, VB, F# or C++ project types.

[testcounter_fs_cs_vb_cpp_njs.zip](https://github.com/Microsoft/nodejstools/files/1605721/testcounter_fs_cs_vb_cpp_njs.zip)

Here is a [video](https://youtu.be/DDfpbjNCrIY) of it happening. I was working with multiple projects for months and haven't noticed this so I think something changed with the newer visual studio version (80% sure). What happens is a nodejs process gets "stuck" and never resumes debug mode. I have disabled the visual studio "break all" option as a temporary workaround.

Joey

**Comment by amcasey** [comment]
Hey @macromaniac,

First off, sorry for the delay - we got distracted with other things and let this go longer than we should have.  Second, thanks for the amazing repro data - the sample project and video were awesome!

Unfortunately (or fortunately, depending on how you look at it), I can't seem to repro this issue, so maybe it was fixed.  I tried following the steps in your video as closely as I could (and repeatedly, in case there was a race) in 15.9.14 and 16.2 Preview and was unable to catch the node console failing to unfreeze.  Are you still seeing this issue?

Thanks,
Andrew
